playground/mocks/idp/idx/data/third-party-idp-without-identify.json

```diff
@@ -0,0 +1,123 @@
+{
```

Comment:
ditto on file naming convention. 
`identify-only-third-party-idps` (sound a bit inaccurate though)

Reply:
yeah, that was one of the reasons why I called them third-party-idp- with/without-identify.json